packages/back-end/src/routers/events/events.router.ts

```diff
@@ -8,7 +8,37 @@ const router = express.Router();
 
 const eventsController = wrapController(rawEventsController);
 
-router.get("/", eventsController.getEvents);
+router.get(
+  "/",
+  validateRequestMiddleware({
+    query: z
+      .object({
+        page: z.string().default("1"),
+        perPage: z.string().default("50"),
```

Comment:
Looks like `zod` can coerce and validate at the same time:
```typescript
z.coerce.number().parse(50)
```
Ref: https://github.com/colinhacks/zod/discussions/330#discussioncomment-5833082